bucket/amp-winoff.json

```
@@ -0,0 +1,19 @@
+{
+    "version": "5.0.1",
+    "description": "An utility for scheduling the shutdown of Windows computers, with several shutdown conditions and fully configurable.",
+    "homepage": "http://www.ampsoft.net/utilities/WinOFF.php",
+    "license": "Freeware",
+    "url": "http://www.ampsoft.net/files/WinOFF.zip",
+    "hash": "af0637ebc5122a93c4da49efb2e29caf999ebbe0c4352a7628732f612893f43d",
+    "bin": "WinOFF.exe",
+    "shortcuts": [
+        [
+            "WinOFF.exe",
+            "AMP WinOFF"
+        ]
+    ],
+    "checkver": "Version ([\\d.]+)",
+    "autoupdate": {
+        "url": "http://www.ampsoft.net/files/WinOFF.zip"
+    }
```

Latest version was 2010

```suggestion
    ]
```

---

Thanks!